tools/migration/funcInfo0.json

```diff
@@ -1,4 +1,4 @@
ton","DPR_PAL_Search_HideCancelButton","DPR_PAL_Search_InitializeSectionLinks","DPR_PAL_SearchAddSearchTermSectionLink","DPR_PAL_Search_AddSectionLink","DPR_PAL_Search_UpdateSectionLink","DPR_PAL_Search_AddSearchTermSectionInfo","DPR_PAL_Search_UpdateSearchTermSectionInfo","DPR_PAL_Search_FixPluralInSearchTermSectionInfo","DPR_PAL_Search_AddCopyPermaLinkElement","DPR_PAL_Search_RemoveCopyPermaLinkElement","DPR_PAL_Search_CreateSectionHeader","searchTipitaka","stopSearch","scrollSearch","DPR_PAL_Search_ScrollSearch","resetvalues","finishSearch","DPR_PAL_Search_ClearSearchResults","pausesall","pausetwo","pausethree","bounce","importXMLs","createTdForMatch","createTables","normalizeLongSearchResult","showonly","atiPause","atiSearchStart","atiSearchOffline","findRegEx","getRegExtSearchString","searchHistoryXML","eraseSearchHistory","sameSearchHistory","simSearchHistory","saveSearchHistory","dictHistoryXML","eraseDictHistory","saveDictHistory","sortaz","sortStrip","comparePaliAlphabet","findSimilarWords","groupBySimilarity","removeDuplicatesFromArray","moveframey","parseDictURLParameters","startDictLookup","pedsearchstart","pedFullTextSearch","dppnsearchstart","dppnFullTextSearch","mlsearchstart","multisearchstart","epdsearchstart","attsearchstart","tiksearchstart","titlesearchstart","paliRootsearchstart","sktsearchstart","sktRootsearchstart","makeUniqueStringForCaseInsensitiveFS","paliXML","toggleDppnTitle","DPPNXML","sktRXML","sktXML","getAtthXML","getTitleXML","getDppnData","displayDictData","openFirstDPRTab","openDPRTab","findDPRTab","findDPRTabs","findDPRTabByLoc","updatePrefs","isDPRTab","giveIDtoTabs","checkLastTab","DPRSidebarWindow","DPRSidebarDocument","closeDPRSidebar","openDPRSidebar","toggleDPRSidebar","setCurrentTitle","closeBrowser","DPRBottomPaneUpdateStyle","DPRShowBottomPane","getSelected","convert","changeConversion","clearboth","sendtoconvert","sendtoPad","clearPad","savePad","moveframex","moveframec","closeCP","openCP","cpFlatten","showHideId","moves","moveanf","scrolldown","getlink","go_anchor","scrollToId","bookmarkXML","eraseBookmark","eraseBookmarks","bookmarkframe","bookmarkxd","bookmarkxn","convertOldBookmarks","saveBookmark","makeDeclensions","outputAnalysis","analyzeword","findmatch","manualCompound","isIndec","isUncomp","outputDef","showShortDef","conciseChange","reanalyze","conjugate","conjugateNoun","conjugateVerb","conjugateIrrNoun","conjugateIrrVerb","conjugateWord","formatConjugation","getConjugation","makeGrammarTerms","typeAndGender","conjugateWords","bv","showBv","citation","bvAlert","translateTextx","translateText","arrangeWords","checkCompatibleNoun","checkCompatibleVerb","makeWord","translateWord","transMod","addPlural","addPhrasePreps","sortLongerDec","stripEnglish","simpleWordTranslation","showAltTable","changeAlt","insertWordByWord","clearText","insertConj","DPR_keypress","createCommands","moveFrame","devO","dalert","ddump","mainInitialize","installGlobalHandlers","triggerUpdateCheck","setupBTForRG","dump"],
```

Comment:
any idea why this file changed?

Reply:
I had renamed the function ` bookmarkXML()` to `getBookmarks()`. This file has been changed during a "Replace all" operation. Is this an issue?

Reply:
no issues - i was just curious. this is used by the tool to migrate our js files to IEFEs in preparation to migration to es6 modules.